test/common/common/utility_test.cc

```diff
@@ -812,11 +812,9 @@ TEST(DateFormatter, FromTime) {
   const SystemTime time1(std::chrono::seconds(1522796769));
   EXPECT_EQ("2018-04-03T23:06:09.000Z", DateFormatter("%Y-%m-%dT%H:%M:%S.000Z").fromTime(time1));
   EXPECT_EQ("aaa23", DateFormatter(std::string(3, 'a') + "%H").fromTime(time1));
-  EXPECT_EQ("", DateFormatter(std::string(1022, 'a') + "%H").fromTime(time1));
```

Comment:
The existing implementation using strftime had a fixed-size 1024 char buffer to hold the formatted string: https://github.com/envoyproxy/envoy/blob/fd79055373e0b127b3d32a062ac93b6bc891fb77/source/common/common/utility.cc#L169 

Since absl::FormatTime returns a std::string, there's no need for a fixed buffer size anymore.

Therefore, there is no reason to test a 1024 character limit.

Reply:
Gotcha!